### PR TITLE
feat: [AXIMST-221] Course unit - use view live and preview links from backend

### PR DESCRIPTION
**Pull request by ihor-romaniuk**
**Description:** use view live and preview links from backend

**Youtrack:** https://youtrack.raccoongang.com/issue/AXIMST-221

### CODE REVIEW

File: src/course-unit/CourseUnit.test.jsx [hunk 1]
```diff
@@ -91,19 +89,18 @@ describe('<CourseUnit />', () => {
     const { open } = window;
     window.open = jest.fn();
     const { getByRole } = render(<RootWrapper />);
+    const { draft_preview_link, published_preview_link } = courseSectionVerticalMock;
```
Comment (ruzniaievdm) [inline src/course-unit/CourseUnit.test.jsx:92]: [nit]: it better to use camelCase style here

Comment (ihor-romaniuk) [reply src/course-unit/CourseUnit.test.jsx]: Agree, changed

**Review by PKulkoRaccoonGang — APPROVED** [review]

**Review by ruzniaievdm — APPROVED** [review]

File: src/course-unit/CourseUnit.test.jsx [hunk 2]
```diff
@@ -89,18 +89,18 @@ describe('<CourseUnit />', () => {
     const { open } = window;
     window.open = jest.fn();
     const { getByRole } = render(<RootWrapper />);
-    const { draft_preview_link, published_preview_link } = courseSectionVerticalMock;
+    const { draft_preview_link: draftPreviewLink, published_preview_link: publishedPreviewLink } = courseSectionVerticalMock;
```
Comment (monteri) [inline src/course-unit/CourseUnit.test.jsx:92]: `This line has a length of 125. Maximum allowed is 120`

Comment (monteri) [reply src/course-unit/CourseUnit.test.jsx:92]: 😄

**Pull request merged**